jlm/rvsdg/region.hpp

```diff
@@ -110,14 +114,12 @@ class RegionResult : public input
 
   ~RegionResult() noexcept override;
 
-protected:
   RegionResult(
       rvsdg::Region * region,
       rvsdg::output * origin,
       structural_output * output,
       std::shared_ptr<const rvsdg::Type> type);
 
-public:
```

Comment:
Same here, deleted constructors should be public.

Reply:
as well, made public